static/app/components/feedback/list/useBulkEditFeedbacks.tsx

```diff
@@ -45,8 +46,13 @@ export default function useBulkEditFeedbacks({deselectAll, selectedIds}: Props)
     organization,
     projectIds: queryView.project,
   });
+  const deleteFeedback = useDeleteFeedback(selectedIds, queryView.project);
 
-  const onToggleResovled = useCallback(
+  const onDelete = () => {
+    deleteFeedback();
+  };
```

Comment:
This also helps by making the identity of `onDelete` stable, so we don't need `useCallback`

```suggestion
  const onDelete = deleteFeedback;
```